tests/integration/helpers.py

```diff
@@ -32,25 +33,39 @@ async def get_unit_url(ops_test: OpsTest, application, unit, port, protocol="htt
     Returns:
         Unit URL of the form {protocol}://{address}:{port}
     """
-    # Sometimes get_unit_address returns a None, no clue why, so looping until it's not
+    # Sometimes get_unit_address returns a None, no clue why.
```

Comment:
This comment might be invalid now.

Reply:
the problem is still there. When we call it on the `get_unit_address` on the `nginx-ingress-integrator` unit, it returns `None`. Possibly it's because the `nginx-ingress-integrator` unit is not yet integrated with a k8s ingress to grab some IP? I'm not sure. But I think the comment should still be there.